server/server.go

```diff
@@ -256,3 +262,188 @@ func (r *rodeServer) ListOccurrences(ctx context.Context, occurrenceRequest *pb.
 		NextPageToken: "",
 	}, nil
 }
+
+func (r *rodeServer) CreatePolicy(ctx context.Context, policyEntity *pb.PolicyEntity) (*pb.Policy, error) {
+	// TODO maybe check if it already exists (if we think a unique name is required)
+
+	log := r.logger.Named("CreatePolicy")
+	// Name fields is a requirement
+	if len(policyEntity.Name) == 0 {
+		return nil, status.Errorf(codes.InvalidArgument, "policy name not provided")
+	}
+
+	// CheckPolicy call will go here
+
+	policy := &pb.Policy{
+		Id:      uuid.New().String(),
+		Version: 1,
+		Policy:  policyEntity,
+	}
+	str, err := protojson.MarshalOptions{EmitUnpopulated: true}.Marshal(proto.MessageV2(policy))
+	if err != nil {
+		return nil, createError(log, fmt.Sprintf("error marshalling %T to json", policy), err)
+	}
+	res, err := r.esClient.Index(
+		rodeElasticsearchPoliciesIndex,
+		bytes.NewReader(str),
+		r.esClient.Index.WithContext(ctx),
+	)
```

Comment:
we should probably refresh the index here, similar to https://github.com/rode/grafeas-elasticsearch/blob/c072ef5eb2ff4f2f1294d281d9158f5e210111ea/go/v1beta1/storage/elasticsearch.go#L765